apps/music/test/marionette/metadata_test.js

```diff
@@ -0,0 +1,73 @@
+/* global require, marionette, setup, suite, test, __dirname */
+'use strict';
+
+var assert = require('assert');
+var Music = require('./lib/music.js');
+var FakeRingtones = require('./lib/fakeringtones.js');
+var FakeControls = require('./lib/fakecontrols.js');
+var ListviewHelper = require('./lib/listviewhelper.js');
+
+marionette('Music player playlist', function() {
+  var apps = {};
+  apps[FakeRingtones.DEFAULT_ORIGIN] = __dirname + '/fakeringtones';
+  apps[FakeControls.DEFAULT_ORIGIN] = __dirname + '/fakecontrols';
+
+  var client = marionette.client({
+    prefs: {
+      'device.storage.enabled': true,
+      'device.storage.testing': true,
+      'device.storage.prompt.testing': true
+    },
+
+    settings: {
+      'lockscreen.enabled': false,
+      'ftu.manifestURL': null
+    },
+
+    apps: apps
+  });
```

Comment:
Ah. I think I see. the fakecontrols and fakeringtones are fake apps that are needed by some of the test.  Most of the stuff from this line above seems like boilerplate marionette setup. Could you refactor this stuff into lib/marionette_setup.js and then replace all this with:

   var client = setup_marionette();

Or something like that?

This is just an idea. I don't feel strongly about making that change.